state_version_integration_test.go

```diff
@@ -120,6 +120,11 @@ func TestStateVersionsCreate(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	jsonState, err := ioutil.ReadFile("test-fixtures/ext-state-version/state.json")
```

Comment:
Youll need to add this fixture, but also can you rename it to `json-state/state.json` ?